frontend/benefit/handler/src/components/applicationReports/ApplicationReports.tsx

```diff
@@ -34,7 +34,7 @@ const ApplicationReports: React.FC = () => {
       <$Heading>{`${t(`${translationsBase}.headings.main`)}`}</$Heading>
 
       <ReportsSection
-        types={['csv', 'pdf']}
+        types={['csv/pdf']}
```

Comment:
`ReportsSection` no longer needs a list of types but just one type instead. Thus I would change this to a `type='csv/pdf'`. Also the prop name `type` isn't the best, it could be for example `exportFileType`.

Reply:
Taken into account in https://github.com/City-of-Helsinki/yjdh/pull/1411/commits/5832bbf825ac90b0bd311a447b5b93b3f3efca16